lib/statesman/adapters/active_record_queries.rb

```diff
@@ -30,6 +34,30 @@ def most_recent_transition_join
 
         private
 
+        def in_state_without_initial(*states)
+          states = states.flatten.map(&:to_s)
+
+          joins(most_recent_transition_join_without_initial).
+            where(states_where(most_recent_transition_alias, states), states).
+            where("#{most_recent_transition_alias}.to_state != ?", initial_state)
```

Comment:
```suggestion

```
You've done a check already that the states do not include `initial` to get to this point. Given that and the fact this uses `JOIN` I can't see a reason to add `!= inital_state`.
Maybe there is a reason I'm missing but if so It'd be great to comment this so the line isn't removed in the future.

Reply:
Good catch! I was just kinda mindless copy/pasting, haha. Fixed in https://github.com/gocardless/statesman/pull/343/commits/4595bdcf7d13c2d4a34aa22e501f72a354376aaa